main.nf

```diff
@@ -207,89 +150,13 @@ process samtools_stats {
 
 }
 
-process samtools_flagstat {
-
-    publishDir "${params.publishdir}/samtools", mode: "copy"
-
-    input:
-    file cram from cram_to_bam_ch_samtools_flagstat
-
-    output:
-    file "*" into samtools_flagstat_ch
-
-    script:
-    """
-    samtools flagstat ${params.sample_id}.qc.${ftype} > ${params.sample_id}.flagstat
-    """
-
-}
-
-process count_variants {
-
-    publishDir "${params.publishdir}/count_variants", mode: "copy"
-
-    input:
-    file vcf from vcf_ch_count_variants
-
-    output:
-    file "*" into count_variants_ch
-
-    script:
-    """
-    count_variants.py \
-        --input_vcf ${vcf} \
-        --output_json ${params.sample_id}.variant_counts.json \
-        --loglevel DEBUG
-    """
-
-}
-
-process bcftools_stats {
-
-    publishDir "${params.publishdir}/bcftools", mode: "copy"
-
-    input:
-    file vcf from vcf_ch_bcftools_stats
-
-    output:
-    file "*" into bcftools_stats_ch
-
-    script:
-    """
-    bcftools stats -f PASS ${vcf} > ${params.sample_id}.pass.stats
-    """
-}
-
-process bcftools_gtcheck {
-
-    publishDir "${params.publishdir}/bcftools", mode: "copy"
-
-    input:
-    file vcf from vcf_ch_bcftools_gtcheck
-    file pst_vcf from pst_vcf_ch_bcftools_gtcheck
-
-    output:
-    file "*.bcftools_gtcheck.txt" into bcftools_gtcheck_ch
-
-    when:
-    params.pst_vcf != null
-
-    script:
-    """
-    bcftools index --tbi ${vcf}
-    bcftools index --tbi ${pst_vcf}
-    bcftools gtcheck --GTs-only 1 --genotypes ${pst_vcf} ${vcf} > ${params.sample_id}.bcftools_gtcheck.txt
-    """
-}
-
 process mosdepth {
 
     publishDir "${params.publishdir}/mosdepth", mode: "copy"
 
     input:
-    file ref_fa from ref_fa_ch_mosdepth
-    file "*" from cram_to_bam_ch_mosdepth
-    file autosomes_bed from autosomes_bed_ch_mosdepth
+    file reference from ref_fa_ch_mosdepth
+    file "*" from cram_bam_ch_mosdepth
     file n_regions_bed from n_regions_bed_ch_mosdepth
```

Comment:
clarify the variable name & all the logic to clarify this file is now gap.txt.gz file

Reply:
n_regions_bed renamed to gap_regionsfile gap_regions from gap_regions_ch_mosdepth